frontend/src/components/shared/Button/Button.tsx

```diff
@@ -0,0 +1,4 @@
+export enum Kind {
+  PRIMARY = "primary",
+  SECONDARY = "secondary",
+}
```

Comment:
This is more a precursor to the Button migration and preparation for a design system. We talked about this for the Alert conversion to have our own enum instead of using BaseWeb. I was migrating the same stuff. Since we only use two versions, I didn't add more, but we can certainly in the future